posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc

```diff
@@ -0,0 +1,124 @@
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not           
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+== Fast iterative development with Liberty Tools
+
+Liberty Tools 23.0.6 are now available for use by Java developers to write cloud-native applications with Liberty.
+
+Liberty Tools support popular integrated development environments (IDEs) such as Eclipse IDE, Visual Studio Code, and IntelliJ IDEA, allowing developers to focus on their code and rapidly iterate by using Liberty's link:/docs/latest/development-mode.html[dev mode] in their preferred IDE. Moreover, Liberty Tools provide editing assistance for developers to easily make changes to the Liberty configuration files, and coding assistance for writing applications using Jakarta EE (9.x and later) and MicroProfile (3.x and later) APIs.
+
+Learn more about link:/docs/latest/develop-liberty-tools.html[Liberty Tools] and download them from different IDE marketplaces:
+
+* link:https://marketplace.eclipse.org/content/liberty-tools[Liberty Tools for Eclipse IDE] 
+* link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Liberty Tools for Visual Studio Code] 
+* link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[Liberty Tools for IntelliJ IDEA] 
+
+Do not hesitate to share your feedback and what you would like to see in future releases via issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].
+
+
+== Build, package, and run easily with Liberty Maven & Gradle plugins
+
+Liberty build plugins enable developers to easily build and package their applications to run on Liberty using the popular Java build tools, Maven and Gradle. The latest versions of Liberty Maven Plugin and Liberty Gradle Plugin are 3.8.2 and 3.6.2 respectively. It's always recommended to use the latest versions of these build plugins to ensure you have the most recent fixes and enhancements. For example, starting with the 3.6 release, the Liberty Gradle Plugin supports running with Gradle 8.x. For more information, see link:https://github.com/OpenLiberty/ci.maven/releases[Liberty Maven Plugin release notes] and link:https://github.com/OpenLiberty/ci.gradle/releases[Liberty Gradle Plugin release notes].
```

Comment:
```suggestion
Liberty build plugins enable developers to easily build and package their applications to run on Liberty using the popular Java build tools, Maven and Gradle. It's always recommended to use the latest versions of these build plugins to ensure you have the most recent fixes and enhancements, Liberty Maven Plugin 3.8.2 and Liberty Gradle Plugin 3.6.2. For example, starting with the 3.6 release, the Liberty Gradle Plugin supports running with Gradle 8.x. For more information, see link:https://github.com/OpenLiberty/ci.maven/releases[Liberty Maven Plugin release notes] and link:https://github.com/OpenLiberty/ci.gradle/releases[Liberty Gradle Plugin release notes].
```

It's easier to parse the version number next to its respective name, rather than having to mentally map the versions to the names, especially for someone skimming the info.